nominal/nominal.py

```diff
@@ -339,3 +348,33 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+
+def draft_checklist(
+    name: str,
+    assignee_email: str,
+    description: str | None = None,
+    default_ref_name: str | None = None,
+) -> ChecklistBuilder:
+    builder = ChecklistBuilder(
+        name=name,
+        assignee_email=assignee_email,
+        description=description,
+        default_ref_name=default_ref_name,
+    )
+    builder._client = get_default_client()
```

Comment:
pydantic would not actually end up setting this field if I tried to set the client in the constructor